trin-core/src/portalnet/overlay_service.rs

```diff
@@ -1152,15 +1301,55 @@ impl<TContentKey: OverlayContentKey + Send, TMetric: Metric + Send>
         Ok(closest_nodes)
     }
 
-    fn send_recursive_findnode(&self, _target: &NodeId) {
-        // TODO: Implement Recursive(iterative) FINDNODE. This is a stub.
+    fn send_iterative_findnode(&mut self, _target: &NodeId) {
+        const DISTANCES_TO_REQUEST_PER_PEER: usize = 1;
```

Comment:
This belongs in a config struct or a `const` at the module level, not in the method.

How did we decide on the value 1? This value is 3 in `sigp/discv5`.